.github/workflows/safari_stable.yml

```diff
@@ -0,0 +1,87 @@
+name: "All Tests: Safari (stable)"
+
+# We never interact with the GitHub API, thus we can simply disable all
+# permissions the GitHub token would have.
+permissions: {}
+
+on:
+  push:
+    branches:
+      - epochs/daily
+      - triggers/safari_stable
+
+env:
+  safaridriver_diagnose: false
```

Comment:
Is this here as a clue that it can be set to true for debugging? If so, can you add a comment?

Reply:
Oh, I should just copy/paste the comment from the prior Azure Pipelines pipeline.